cyhy/mailer/cli.py

```diff
@@ -793,9 +795,13 @@ def send_cyhy_reports(
             if not to_emails:
                 continue
 
-            # Grab the email address of the appropriate CSA.
+            # Grab the email address of the appropriate CSA for non-Federal and
+            # non-International entities.
             csa_email_address = None
-            if csa_emails is not None:
+            if csa_emails is not None and request["agency"].get("type") not in [
+                "FEDERAL",
+                "INTERNATIONAL",
+            ]:
```

Comment:
This list of agency types should be a constant, particularly since we use it twice.

Reply:
Good thinking - please see https://github.com/cisagov/cyhy-mailer/pull/107/commits/f768699e7aba8d979a2c961a6f575c617c68814e.